test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,56 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind an imported Node.js application to PostgreSQL database in the following order: CSV, DB, App and SBR
```

Comment:
Unclear that is the scenario about, please rephrase. What are we testing here?

Reply:
This is same as other scenarios in the list, just adding CSV step in it.

Reply:
Removed gratuitous scenario.